R/utils_plotting.R

```diff
@@ -349,9 +367,6 @@ NULL
     if (is.null(edge_size_by)) {
```

Comment:
Does this break `plotTree`? Commit: https://github.com/microbiome/miaViz/commit/6bd92a8e9565749db90dad43f76a7c54043101f5

Reply:
Why should it?

Reply:
..that `layout` is removed. The comment is in wrong place

Reply:
The problem was that edges were plotted with default layout. --> two different trees were plotted. PR: https://github.com/microbiome/miaViz/pull/25

Reply:
Keep it in mind, if you see a problem. I am not sure, why added it in the first place, but there must have been a reason. Maybe you should look into again with this mind. I don't remember the original issue with the two trees, but I think it worth investigating.

Reply:
**This was the problem**
git checkout 5cace72fb92072fd79375a6b8b2aacbb063ad8d2
```
data("GlobalPatterns")
se <- GlobalPatterns 
se <- agglomerateByRank(se, rank = "Phylum")
plotRowTree(se,
            edge_colour_by = "Kingdom", layout="ellipse")
```
  
![image](https://user-images.githubusercontent.com/60338854/127702988-56c73b7a-a4a8-4e01-8e15-cc007c53e745.png)

 
**Solved in master:**
![image](https://user-images.githubusercontent.com/60338854/127703012-b37ae3bb-69e2-49ac-9380-1bd2c1c549a3.png)

 
**order_descending:**
```
 Error in .get_edge_args(edge_colour_by, edge_size_by, alpha = line_alpha,  : 
  unused argument (layout = layout) 
```

Reply:
So the layout did not propagate or, was it not used by the `ggtree` function?

Reply:
plotRowTree --> .plot_row_column_tree --> .tree_plotter --> .plot_tree_edges:

gets layout inside plot_out argument and as layout argument. 

The problem was that `layout` inside `plot_out` did not work here were edges are plotted. This part needed its own `layout` to work
![image](https://user-images.githubusercontent.com/60338854/127704495-54ccc695-e174-4947-9d13-eb8eb3a87e29.png)

Reply:
```
 Error in .get_edge_args(edge_colour_by, edge_size_by, alpha = line_alpha,  : 
  unused argument (layout = layout) 
```

This should have errored already for you, since the you removed the layout argument from the function definition, but not from the call in line 1244.

So I fixed it by adding the layout argument again, but no using it.

However, jsut to get back to the original question: `geom_tree` with using the layout argument produced a different tree, than was already set in plot_out, correct? So I think this would be a bug in `ggtree`. Do you want to report it?

Reply:
```
library("mia")
data("GlobalPatterns")
se <- GlobalPatterns 

se <- agglomerateByRank(se, rank = "Phylum", agglomerateTree = TRUE)
tree <- rowTree(se)

plot_out <- ggtree(tree,
                   layout = "ellipse")

plot_out + geom_tree()
```

![image](https://user-images.githubusercontent.com/60338854/127706320-83b2782f-dfd2-4416-9c86-e7e6e14fdd60.png)

Looks like it, yes I can report it

Reply:
I think you have to simplify the example more. Just save the tree to file and upload as zip to the issue